autovalet_description/urdf/autovalet.urdf.xacro

```diff
@@ -7,7 +7,7 @@
 	<xacro:include filename="$(find velodyne_description)/urdf/VLP-16.urdf.xacro"/>
 
 	<sensor_d435 parent="front_bumper_link" name="frontCamera" >
-	    <origin xyz="-0.305 0.02 0.91" rpy="0 0 0" />
+	    <origin xyz="-0.305 0.02 0.91" rpy="0 0.174 0" />
```

Comment:
this is only 10 deg fyi, not 45 like you said in the description

Reply:
changed my description.